xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -99,15 +99,16 @@ static const dxva2_mode_t dxva2_modes[] = {
     { "HEVC / H.265 variable-length decoder, main",   &D3D11_DECODER_PROFILE_HEVC_VLD_MAIN,   AV_CODEC_ID_HEVC },
     { "HEVC / H.265 variable-length decoder, main10", &D3D11_DECODER_PROFILE_HEVC_VLD_MAIN10, AV_CODEC_ID_HEVC },
 
+    /* VP9 */
+    { "VP9 VLD, Profile 0", &D3D11_DECODER_PROFILE_VP9_VLD_PROFILE0, AV_CODEC_ID_VP9 },
```

Comment:
I'm getting a compile error on Windows 7:

```
DXVA.cpp(103): error C2065: 'D3D11_DECODER_PROFILE_VP9_VLD_PROFILE0': undeclared identifier
```

Reply:
My `d3d11.h` header says:

```c++
/* File created by MIDL compiler version 8.00.0603 */
```

Reply:
You have to update Windows SDK